core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -78,6 +78,12 @@ jQuery.extend( jQuery.fn.dataTableExt.oSort, {
     }
 } );
 
+jQuery.extend( jQuery.fn.dataTableExt.ofnSearch, {
+    "appid-numeric": function ( a ) {
+        return a.replace(/[\r\n]/g, " ").replace(/<.*?>/g, "");
```

Comment:
Refer to `jquery.dataTables.1.10.4.min.js`. I'd like to change it to better style if there's any :)